src/test/java/teammates/test/cases/action/StudentCourseJoinActionLegacyTest.java

```diff
@@ -0,0 +1,59 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
+import teammates.logic.core.StudentsLogic;
+import teammates.ui.controller.StudentCourseJoinAction;
+
+/**
+ * Test Course Join Link for Unregistered Student
```

Comment:
How about "To test legacy course join link for unregistered students."? The other words are unneccessarily capitalised.